src/knex-store.ts

```diff
@@ -166,6 +166,17 @@ function knex_store(this: any, options: Options) {
     }
   )
 
+  seneca.add(
```

Comment:
use seneca.message for async actions

Reply:
Humm, what would be the difference then between seneca.message and seneca.add, I tried looking in the documentation but I don't know if it's right.

Reply:
Anyway, done